sdk/textanalytics/ai-text-analytics/samples/javascript/recognizePii.js

```diff
@@ -25,45 +25,45 @@ async function main() {
   const client = new TextAnalyticsClient(endpoint, new AzureKeyCredential(apiKey));
 
   const textOnePii = "My phone number is 555-5555";
-  const textNoPHI =
-    "FIFA is a non-profit organization which describes itself as an international governing body of association football.";
+  const textNoPHI = "His EU passport number is X65097105";
   const textMultiplePIIs = "Patient name is Joe and SSN is 859-98-0987";
 
   const [result] = await client.recognizePiiEntities([textOnePii]);
 
   if (!result.error) {
     console.log(
-      `The redacted text is "${result.redactedText}" and found the following PII entities`
+      `The redacted text is \"${result.redactedText}\" and found the following PII entities`
     );
     for (const entity of result.entities) {
-      console.log(`\t- "${entity.text}" of type ${entity.category}`);
+      console.log(`\t- \"${entity.text}\" of type ${entity.category}`);
     }
   }
 
-  console.log(`There are no PHI entities in this text: ${textNoPHI}`);
-  const [resultWithPHI] = await client.recognizePiiEntities(
-    [{ id: "0", text: textNoPHI, language: "en" }],
-    { domainFilter: PiiEntityDomainType.PROTECTED_HEALTH_INFORMATION }
-  );
+  console.log(`There are no PHI entities in this text: \"${textNoPHI}\"`);
+  const [resultWithPHI] = await client.recognizePiiEntities([textNoPHI], "en", {
+    domainFilter: PiiEntityDomainType.PROTECTED_HEALTH_INFORMATION
+  });
   if (!resultWithPHI.error) {
-    console.log(`Also there is nothing to redact: ${resultWithPHI.redactedText}`);
+    console.log(`Also there is nothing to redact: \"${resultWithPHI.redactedText}\"`);
     assert(resultWithPHI.entities.length === 0, "did not expect any entities but got some");
   }
 
   console.log(`But there are other PII entities in that text:`);
   const [resultWithoutPHI] = await client.recognizePiiEntities([textNoPHI]);
   if (!resultWithoutPHI.error) {
     for (const entity of resultWithoutPHI.entities) {
-      console.log(`\t- "${entity.text}" of type ${entity.category}`);
+      console.log(`\t- \"${entity.text}\" of type ${entity.category}`);
```

Comment:
Do you really need to escape these quotes within a template string literal?

Reply:
Nope, I got confused for some reason. Fixed.